src/beliefpropagation.jl

```diff
@@ -99,237 +99,54 @@ function update_all_mts(
 end
 
 """
-given two flat networks psi and psi0, calculate the ratio of their contraction centred on the the subgraph containing v. The message tensors should be formulated over psi
-Link indices between psi and psi0 should be consistent so the mts can be applied to both
+Given a subet of vertices of a given Tensor Network and the Message Tensors for that network, return a Dictionary with the involved subgraphs as keys and the vector of tensors associated with that subgraph as values
+Specifically, the contraction of the environment tensors and tn[vertices] will be a scalar.
 """
-function get_single_site_expec(
-  tn::ITensorNetwork,
-  subgraphs::DataGraph,
-  tnO::ITensorNetwork,
-  v;
-  contraction_sequence::Function=tn ->
-    ITensorNetworks.contraction_sequence(tn; alg="optimal"),
-)
-  subgraph = find_subgraph(v, subgraphs)
-  num_tensors_to_contract = ITensor[]
-  denom_tensors_to_contract = ITensor[]
-  for k in neighbors(subgraphs, subgraph)
-    push!(num_tensors_to_contract, subgraphs[k => subgraph])
-    push!(denom_tensors_to_contract, subgraphs[k => subgraph])
-  end
-  for vertex in vertices(subgraphs[subgraph])
-    push!(num_tensors_to_contract, tnO[vertex])
-    push!(denom_tensors_to_contract, tn[vertex])
-  end
-  numerator = ITensors.contract(
-    num_tensors_to_contract; sequence=contraction_sequence(num_tensors_to_contract)
-  )[]
-  denominator = ITensors.contract(
-    denom_tensors_to_contract; sequence=contraction_sequence(denom_tensors_to_contract)
-  )[]
-  return numerator / denominator
-end
-
-"""
-given two flat networks psi and psi0, calculate the ratio of their contraction centred on the subgraph(s) containing v1 and v2. The message tensors should be formulated over psi.
-"""
-function get_two_site_expec(
-  psi::ITensorNetwork,
-  subgraphs::DataGraph,
-  psiO::ITensorNetwork,
-  v1,
-  v2;
-  contraction_sequence::Function=tn -> contraction_sequence(tn; alg="optimal"),
-)
-  subgraph1 = find_subgraph(v1, subgraphs)
-  subgraph2 = find_subgraph(v2, subgraphs)
-  num_tensors_to_contract = ITensor[]
-  denom_tensors_to_contract = ITensor[]
-
-  if (subgraph1 == subgraph2)
-    connected_subgraphs = neighbors(subgraphs, subgraph1)
-    for k in connected_subgraphs
-      push!(num_tensors_to_contract, subgraphs[k => subgraph1])
-      push!(denom_tensors_to_contract, subgraphs[k => subgraph1])
-    end
+#CAN SWITCH DIR TO GET OUTGOING ENVIRONMENT BUT THIS WON'T BE QUITE RIGHT IF WE ARE DOING GENERAL BELIEF PROPAGATION
```

Comment:
This should get moved above the docstring (also please make it normal-cased, not upper case).